eq-author/src/components/Collapsible/ContentCollapsible.js

```diff
@@ -0,0 +1,165 @@
+import React, { useState } from "react";
```

Comment:
How come you've chosen to make this into a new file instead of editing the original to accept a varients prop to change it's appearence?